doc/source/whatsnew/v0.19.0.txt

```diff
@@ -228,25 +228,25 @@ Previous behaviour:
    0  2  1  2
    1  5  4  5
 
-The first ``a`` column contains the same data as the second ``a`` column, when it should have
+The first ``a`` column contained the same data as the second ``a`` column, when it should have
 contained the values ``[0, 3]``.
 
 New behaviour:
 
-.. ipython :: python
+.. ipython:: python
 
-   In [2]: pd.read_csv(StringIO(data), names=names)
+   pd.read_csv(StringIO(data), names=names)
 
 
 .. _whatsnew_0190.enhancements.read_csv_categorical:
 
-:func:`read_csv` supports parsing ``Categorical`` directly
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+``read_csv`` supports parsing ``Categorical`` directly
```

Comment:
It's 'not', because otherwise the linking from the contents overview at the top doesn't work (that was the reason I changed it, rather than consistency, but in any case, we should be consistent in not using it in headers :-))